js/templates.js

```diff
@@ -1,185 +1,497 @@
-(function() {
```

Comment:
You forgot to commit the changed handlebar files?

Reply:
@nickvergessen 

> I'm not sure about the change of hiding the message on default.
> For me that basically means the important info is always one click further away than necessary. Especially when (like in your example and also our instance) the announcements are rather short (1-10 lines) it's totally fine to show them. In most usecases I saw only the last one matters anyway.

Agreed! Good point, adjusted in kind.

Also added in the handlebars, oops.